### PR TITLE
UNB3M tropospheric model and iono/tropo Python bindings

### PR DESCRIPTION
**Pull request by nsirola**
Added bindings wrapping the existing iono and tropo correction functions, needed for SPP development in gnss-analysis
@gsmcmullin

### CODE REVIEW

File: python/swiftnav/iono.pxd [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  double calc_ionosphere(const gps_time_t *t_gps,
+                       double lat_u, double lon_u,
+                       double a, double e,
+                       const ionosphere_t *i);
+
+  ctypedef struct ionosphere_t:
+    double a0, a1, a2, a3;
+    double b0, b1, b2, b3;
+
+  ctypedef struct gps_time_t:
```
Comment (ljbade) [inline python/swiftnav/iono.pxd]: Isn't this defined elsewhere already?

File: python/swiftnav/iono.pyx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  Returns
+  -------
+  out : float
+    Ionospheric delay in meters
+
+  """
+  assert len(alpha) == 4 and len(beta) == 4, "Ionospheric parameter vectors alpha and beta must have 4 elements each."
+  cdef gps_time_t t
+  t.tow = tow
```
Comment (ljbade) [inline python/swiftnav/iono.pyx]: Any reason for not passing a `GpsTime` object?

**Comment by ljbade** [comment]
So this PR is identical to the the two in private?

**Comment by nsirola** [comment]
Yes these are the same commits in fact.

**Comment by ljbade** [comment]
OK Ill merge then.

**Pull request merged**